.gitignore

```diff
@@ -6,10 +6,10 @@ automated-release-build
 etc/dev.yml
 
 # Dependencies
-/node_modules
+/node_modules/
 
-# Production
```

Comment:
stop ignoring /build/ because it contains needed build files, not build artifacts, and start ignoring the actual artifacts dir /dist/